README.md

```diff
@@ -36,9 +36,10 @@ may file one [here](https://github.com/NVIDIA/spark-rapids/issues/new/choose).
 
 ## Releases
 
+For a list of releases please visit [here](https://github.com/NVIDIA/spark-rapids/releases/)
 | Version | Description |
 |---------|-------------|
-
+| [0.1.0](https://github.com/NVIDIA/spark-rapids/releases/tag/v0.1.0)| First release|
```

Comment:
This does not point to the documentation.  Can we add something around here pointing to 

https://nvidia.github.io/spark-rapids/

Reply:
Done

Reply:
Agree with the comments above.  Let's add a link to our docs pages https://nvidia.github.io/spark-rapids/ at the top of the doc: 

`NOTE: For the latest stable [README.md](https://github.com/nvidia/spark-rapids/blob/main/README.md) ensure you are on the main branch.  The RAPIDS Accelerator for Apache Spark provides a set of plugins for Apache Spark that leverage GPUs to accelerate processing via the RAPIDS libraries and UCX.  Documentation on the current release can be found at https://nvidia.github.io/spark-rapids/ 
`
We can remove the releases section altogether.  We will add that back to our https://nvidia.github.io/spark-rapids/ pages at a future date.  

Note that this PR refers to main as the primary repository, which is the direction we will move in.

Reply:
The verbiage on the last sentence `Documentation on the current release can be found at https://nvidia.github.io/spark-rapids/` may have to be changed because I would expect to see a release notes section which that link doesn't have. 

That link, as I understand, is an over arching documentation for our project not just that release? So may be have something more general, like, `Documentation on Rapids-Spark can be found at https://nvidia.github.io/spark-rapids/`?

Reply:
The idea is to have release notes when we have future releases.  For the 0.1 release we just have the documentation.

Reply:
gotcha... change made. PTAL

Reply:
Please remove the code formatting around the added text, so the URL links render.

Reply:
I have made the change. Please take another look